typing/predef.ml

```diff
@@ -51,6 +51,7 @@ and ident_extension_constructor = ident_create "extension_constructor"
 and ident_floatarray = ident_create "floatarray"
 and ident_lexing_position = ident_create "lexing_position"
 and ident_atomic_loc = ident_create "atomic_loc"
+and ident_code = ident_create "expr"
```

Comment:
Not super vital, but why different names? code vs. expr?

Reply:
We initially wanted to use `t code`, but `expr` is more accurate because it's only expressions being quoted and not patterns/types. There are a few references to `code` internally and I could replace all of them with `expr`. Do you think it would be a worthwhile change to make?

Reply:
I actually think `t code` is unambiguous, because (I think?) language categories don't mix even across stages? That is, a type `t` in stage 1 can be embedded into stage 0 as `<[ t ]>`, which is type-like (in the sense of having a jkind). We will never have an image of `t` in stage 0 available as an expression? That means, whenever in stage 0 we see an expression of type `_ code`, we know it must corresponds to an expression in stage 1 or so.